spec/ddtrace/profiling/ext/cthread_spec.rb

```diff
@@ -32,6 +37,15 @@
     # Kill any spawned threads
     after { thread.kill if instance_variable_defined?(:@thread_started) && @thread_started }
 
+    def simulate_missing_instrumentation
```

Comment:
I see how this works;what do you think about defining it as a `thread_class_without_instrumentation`?

That way we could compose it in by substituting the class instead of using `before`? Makes it easier to compose and control conditional test branches when there are fewer `before` statements.

Reply:
Updated! I went with `thread_class_missing_instrumentation` since I "without" seemed to suggest that we never added it.